examples/Tilt-Compensated Compass Example

```diff
@@ -0,0 +1,77 @@
+## Tilt Compensated Compass using Accelerometer and Magnetometer
+
+# # Equations from st manual, "Computing tilt measurement and tilt-compensated eCompass" by Andrea Vitali
+   # https://www.st.com/resource/en/design_tip/dm00269987-computing-tilt-measurement-and-tiltcompensated-ecompass-stmicroelectronics.pdf
+    
+   # for Phi
+      #  Roll: phi = atan2(Gy, Gz) # for stability ,can be substituted with Gz+Gx*alpha (with alpha=0.01-0.05)
+   # for Theta
+      # Gz2 = Gy * Sin( Phi ) + Gz * Cos( Phi )
+      # Pitch: Theta = Atan( -Gx / Gz2)
+   # for Psi or "Yaw" (The Tilt Compensated Heading); Bx,By,Bz are magnetometer readings
+      # By2 = Bz * Sin( Phi ) – By * Cos( Phi )
+      # Bz2 = By * Sin( Phi ) + Bz * Cos( Phi )
+      # Bx3 = Bx * Cos( Theta ) + Bz2 * Sin( Theta )
+      # Yaw: Psi = Atan2( By2 , Bx3)
+###############################################
```

Comment:
The comments needs to be aligneg, pretty sure the CI will reject this,. so it will be as

```python
# # Equations from st manual, "Computing tilt measurement and tilt-compensated eCompass" by Andrea Vitali
# https://www.st.com/resource/en/design_tip/dm00269987-
# 
# for Phi
```